Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIWidget.cxx

```diff
@@ -63,8 +67,52 @@ void vtkSlicerROIWidget::CreateDefaultRepresentation(
   rep->UpdateFromMRML(nullptr, 0); // full update
 }
 
+//-------------------------------------------------------------------------
+bool vtkSlicerROIWidget::ProcessMouseMove(vtkMRMLInteractionEventData* eventData)
+{
+  vtkMRMLMarkupsNode* markupsNode = this->GetMarkupsNode();
+  vtkSlicerMarkupsWidgetRepresentation* rep = this->GetMarkupsRepresentation();
+  if (!rep || !markupsNode || !eventData)
+    {
+    return false;
+    }
+
+  int state = this->WidgetState;
+  // Process the motion
+  // Based on the displacement vector (computed in display coordinates) and
+  // the cursor state (which corresponds to which part of the widget has been
+  // selected), the widget points are modified.
+  // First construct a local coordinate system based on the display coordinates
+  // of the widget.
+  double eventPos[2]
+    {
+    static_cast<double>(eventData->GetDisplayPosition()[0]),
+    static_cast<double>(eventData->GetDisplayPosition()[1]),
+    };
+
+  if (state == WidgetStateScale)
+    {
+    bool symmetricScale = eventData->GetModifiers() & vtkEvent::ShiftModifier;
```

Comment:
At this point, we should not check the modifier anymore, because it would prevent dynamic remapping of GUI->widget events. Instead, Shift+MouseMove should be mapped to a new widget event trio (something like WidgetStateSymmetricScale, WidgetStateSymmetricScaleStart, WidgetStateSymmetricScaleEnd).